app/javascript/components/side-bar/Sidebar.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from 'react';
+import Instagram from '../../assets/instagram-icon.svg';
+import Facebook from '../../assets/facebook.svg';
+import sidebar from './sidebar.scss';
+
+const Sidebar = ({isOpen, actionClose}) => {
+  const [taxonomies, setTaxonomies ] = useState([]);
+
+  useEffect(() => {
+    document.body.style.overflow = `${isOpen ? 'hidden' : 'unset'}`
+  },[isOpen] )
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  const fetchData = () => {
+    fetch('/solidus/api/taxonomies.json', {
+      headers: {
+        'Content-Type': 'application/json',
+        'Authorization': 'Bearer 49d9878bc37c2fb5c8e363224259f1a2f520f57cebf69c3e'
```

Comment:
change this to use ENV vars

Reply:
I solution this extracting the value front solidus and rendering it in index.html.erb